# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/worklet/WorkletGlobalScope.cpp
# Commit: 5278e40b80c3
# Full Hash: 5278e40b80c3c4aa642a24a1ff5a50247b9b3126
# Author: Tom Ritter <tom@mozilla.com>
# Date: 2022-11-28 15:41:50
# Regressor Bug: 1778510
# File Overlap Count: 1
# Description:
#   Bug 1778510: Add RTPCallerType to GlobalObject r=asuth
#   
#   This centralizes the logic in one place.
#   
#   In order to do this, we will need to check the principal
# ==============================================================================

diff -r dba220581d93 -r 5278e40b80c3 dom/worklet/WorkletGlobalScope.cpp
--- a/dom/worklet/WorkletGlobalScope.cpp	Mon Nov 28 04:21:26 2022 +0000
+++ b/dom/worklet/WorkletGlobalScope.cpp	Mon Nov 28 04:21:26 2022 +0000
@@ -76,6 +76,10 @@
   return mImpl->ShouldResistFingerprinting();
 }
 
+bool WorkletGlobalScope::IsSystemPrincipal() const {
+  return mImpl->IsSystemPrincipal();
+}
+
 void WorkletGlobalScope::Dump(const Optional<nsAString>& aString) const {
   WorkletThread::AssertIsOnWorkletThread();
 